### DISCUSSION TITLE
changing font size in legend of ResponsivePie

### DISCUSSION THREAD

**Discussion by JoeyFaulkner** [plouc/nivo#1757] [Q&A] [answered]
Hey there! really love this project, would appreciate any help available. 

We are using ResponsivePie in our project but we're finding the text is far too small in the legend. 
![image](https://user-images.githubusercontent.com/1977969/131698441-f089f6dc-52bd-4b13-8f58-eb5c9f92d6f9.png)

I have extensively searched the code and the documentation for any way of changing the fontsize in a legend but come to nothing. Is there any way around this, or anything i've missed?

Thanks!

**Comment by wyze** [comment, marked as answer]
You should be able to set it on the theme `<ResponsivePie theme={{ legends: { text: { fontSize: 20 } } }} />`.

**Reply by JoeyFaulkner** [reply]
thanks so much!

**Reply by shellyscheng** [reply]
is there a way to change font family?

**Reply by papahuangson** [reply]
@shellyscheng Same as above but with `fontFamily` property. Also be sure to target the proper theme property. 

For example, you can have legends defined in `legends` and/or one of the `axes`.

```
theme={{
  legends: {
    text: {
      fontFamily: 'Comic Sans',
    },
  },
  axis: {
    legend: {
      text: {
        fontFamily: 'Comic Sans',
      },
    },
  },
}}
```
